api/src/test/java/org/apache/iceberg/util/TestExceptionUtil.java

```diff
@@ -1,170 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package org.apache.iceberg.util;
-
-import java.io.IOException;
-import org.assertj.core.api.Assertions;
-import org.junit.Assert;
-import org.junit.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class TestExceptionUtil {
```

Comment:
Why remove the test if the tested code is still in the codebase? Even if deprecated, it'd be nice to validate that it works. These tests do not look too expensive to run :thinking:

Reply:
Happy to add them back!

Reply:
thx :)

Reply:
nit: update the PR'd summary line?

Reply:
Updated. I'm going to add a note about removing in 1.0 as well.